Svc/FileManager/test/ut/FileManagerTester.cpp

```diff
@@ -363,6 +363,169 @@ void FileManagerTester ::fileSizeFail() {
     this->assertFailure(FileManager::OPCODE_FILESIZE);
 }
 
+void FileManagerTester ::listDirectorySucceed() {
+#if defined TGT_OS_TYPE_LINUX || TGT_OS_TYPE_DARWIN
+    // Remove test_dir and create it with some files
+    this->system("rm -rf test_dir");
+    this->system("mkdir test_dir");
+    this->system("touch test_dir/file1.txt");
+    this->system("touch test_dir/file2.txt");
+    this->system("touch test_dir/file3.dat");
+#else
+    FAIL();  // Commands not implemented for this OS
```

Comment:
My preference is to use `SKIP()` here, because an unimplemented unit test should not result in a failure.